### PR TITLE
Cleanup github actions

### PR DESCRIPTION
**Pull request by FloGo3308**
Just making it a little more consistent.
And `    - name: Set up JDK 9` was wrong cuz it runs on java 8, not 9

### CODE REVIEW

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 9
+    - name: Set up JDK 8
```
Comment (scorbett123) [inline .github/workflows/run_tests.yml:15]: Sorry, I obviously mistyped.

**Review by scorbett123 — APPROVED** [review]

**Review by ZacSharp — APPROVED** [review]
Not sure about the renamed tasks but definitely a good thing in general.

**Comment by FloGo3308** [comment]
> Not sure about the renamed tasks but definitely a good thing in general.

What would you prefer then? I changed the build action to match the style of the test action becouse the build action still contained some stuff from the template.

Some questions:
- should I change the file name of the build action too?
- should I change the name of the build step to `Executing build` like the `Executing tests` the test action uses?(`Executing build` doesn't sound right though)
- why is the name(line 2) of the test action called `Tests` but the job(line 9) called `test`(without a s at the end)?(just a minor detail)

**Comment by ZacSharp** [comment]
>     * should I change the file name of the build action too?

I think if the name doesn't mention gradle, then the name should not do it as well.
>     * should I change the name of the build step to `Executing build` like the `Executing tests` the test action uses?(`Executing build` doesn't sound right though)

That's sounds wrong as you said. I think this one is the only name using a progressive form.
>     * why is the name(line 2) of the test action called `Tests` but the job(line 9) called `test`(without a s at the end)?(just a minor detail)

Don't know, if you think it should be changed this is the pr for it.

As I said I'm not sure about the renames, which in this case means me having doubts about pretty much every line of this answer.
I'll send it anyway because I don't want to leave you without any answer at all.

File: .github/workflows/workflow_test.yml [hunk 2]
```diff
@@ -12,15 +11,14 @@ jobs:
...
       with:
         java-version: '8'
         distribution: 'adopt'
-        
+
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
       
-    - name: Executing tests
+    - name: Executing test
```
Comment (scorbett123) [inline .github/workflows/workflow_test.yml:23]: Change these both to the plural tests.

**Review by scorbett123 — CHANGES_REQUESTED** [review]

File: .github/workflows/workflow_test.yml (hunk 2 above)
Comment (FloGo3308) [reply .github/workflows/workflow_test.yml:23]: wdym with plural?

Comment (scorbett123) [reply .github/workflows/workflow_test.yml:23]: tests not test

**Pull request closed without merging**